plugin/aws/ec2Instance.go

```diff
@@ -217,7 +217,13 @@ func (inst *ec2Instance) Exec(ctx context.Context, cmd string, args []string, op
 		return nil, fmt.Errorf("No public interface found for %v", inst)
 	}
 
+	var identityfile string
+	if name, ok := meta["KeyName"]; ok {
+		identityfile = ("~/.ssh/" + name.(string) + ".pem")
```

Comment:
We should fallback to ssh-agent if the file doesn't exist.

Reply:
Done.